.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+on: [pull_request]
+name: changed files lint
+jobs:
+  scan:
+    name: npm lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+          fetch-tags: true
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '22.11.0'
```

Comment:
I wonder if there is a way to tie this to Dockerfile like we do here: https://github.com/DataBiosphere/duos-ui/blob/d281e1effa6233d711bcff89a0fed3ab9e423def/docs/getting-started.md?plain=1#L73

Reply:
However, we reference node directly in other actions as well, so definitely not a blocker.

Reply:
Should we reference the `develop` branch or the branch the PR is on?  I'll look at extracting the file into an environment variable and l

Reply:
Huzzah!  Figured this out!

Reply:
Just in time for the node update in https://github.com/DataBiosphere/duos-ui/pull/2789 😄

Reply:
Do you think it makes sense to migrate the other workflows that use node-version to this methodology?

Reply:
> Do you think it makes sense to migrate the other workflows that use node-version to this methodology?

I do, but that seems like a new ticket to me.

Reply:
Per standup, I've added the changes to this PR.